src/pages/VaultsPage/DepositModal.tsx

```diff
@@ -35,10 +36,26 @@ export default function DepositModal({
 }) {
   const classes = useStyles();
   const history = useHistory();
+  const [balance, setBalance] = useState<any>();
```

Comment:
```suggestion
  const [max, setMax] = useState<any>();
```
this indicates what the value is used for here better, IMO